app/config/MicroserviceAppConfig.scala

```diff
@@ -103,6 +107,14 @@ class MicroserviceAppConfig @Inject()(servicesConfig: ServicesConfig, val config
   override lazy val desEnvironment: String = loadConfig(s"$desBase.environment")
   override lazy val desToken: String = loadConfig(s"$desBase.authorization-token")
 
+  private def itsaIncomeSourceBase =
+    if (FeatureSwitching.isEnabled(featureswitch.HIPItsaIncomeSource, configuration)) "microservice.services.stub-itsa-income-source"
```

Comment:
Did you want this feature switch to be the "use stub" feature switch, otherwise i'm not sure what this is for.